test/test.js

```diff
@@ -182,6 +183,15 @@ function expectCreatedShot(driver, creator) {
   });
 }
 
+function verifyShotUrl(shotUrl) {
+  const restUrl = shotUrl.substr(backend.length);
+  const hostname = new URL(backend).hostname;
+  const regex = new RegExp(`^/[^/]+/${hostname}$`)
+  if (!regex.test(restUrl)) {
```

Comment:
Since we're already using `URL` here, it might be more consistent and more readable to get rid of `restUrl` and `regex` and instead do something like,

```
let url = new URL(shotUrl);
if (!url.pathname.endsWith(hostname)) {
```